server/app/views/applicant/TIAccountSettingsView.java

```diff
@@ -0,0 +1,144 @@
+      ApplicantPersonalInfo personalInfo,
+      Http.Request request,
+      Messages messages,
+      Long currentTisApplicantId) {
+    HtmlBundle bundle =
+        layout
+            .getBundle(request)
+            .setTitle("CiviForm")
+            .addMainContent(
+                h1(tiGroup.getName()).withClasses(BaseStyles.TI_HEADER_BAND_H1),
+                TrustedIntermediaryDashboardView.renderTabButtons(
+                    messages, baseUrl, TrustedIntermediaryDashboardView.TabType.ACCOUNT_SETTINGS),
+                div(
+                    renderSubHeader(messages.at(MessageKey.HEADER_ACCT_SETTING.getKeyName()))
+                        .withClasses(BaseStyles.TI_HEADER_BAND_H2),
+                    div(
+                            h3(messages.at(MessageKey.TITLE_ORG_MEMBERS.getKeyName()))
+                                .withClass("mt-8"),
+                            renderTIMembersTable(tiGroup, messages))
+                        .withClass("px-20")))
+            .addMainStyles("bg-white");
+
+    Http.Flash flash = request.flash();
+    if (flash.get("error").isPresent()) {
+      LoggerFactory.getLogger(TrustedIntermediaryGroupListView.class)
+          .info(request.flash().get("error").get());
+      bundle.addToastMessages(
+          ToastMessage.errorNonLocalized(flash.get("error").get()).setDuration(-1));
+    }
+    return layout.renderWithNav(request, personalInfo, messages, bundle, currentTisApplicantId);
+  }
+
+  private DivTag renderTIMembersTable(TrustedIntermediaryGroupModel tiGroup, Messages messages) {
+    return div(
+        table()
+            .withData("testid", "org-members-table")
+            .withClasses("usa-table", "usa-table--striped", "w-5/6")
+            .with(renderGroupTableHeader(messages))
+            .with(
+                tbody(
+                    each(
+                        tiGroup.getTrustedIntermediaries().stream()
+                            .sorted(Comparator.comparing(AccountModel::getApplicantName))
+                            .collect(Collectors.toList()),
+                        this::renderTIRow))));
+  }
+
+  private TheadTag renderGroupTableHeader(Messages messages) {
+    return thead(
+        tr().with(
+                th(messages.at(MessageKey.NAME_LABEL.getKeyName()))
+                    .withScope("col")
+                    .withData("testid", "org-members-name")
+                    .withClass("w-1/3"))
+            .with(
+                th(messages.at(MessageKey.EMAIL_LABEL.getKeyName()))
+                    .withScope("col")
+                    .withData("testid", "org-members-email")
+                    .withClass("w-2/5"))
+            .with(
+                th(messages.at(MessageKey.ACCT_STATUS_LABEL.getKeyName()))
+                    .withScope("col")
+                    .withData("testid", "org-members-status")
+                    .withClass("w-1/5")));
+  }
+
+  private TrTag renderTIRow(AccountModel ti) {
```

Comment:
**suggestion:** I know this is copied over, but similar comment to above on making the casing consistent. May be worth changing it to renderTiRow?

Reply:
I don't like using sentence-case for a 2-letter acronym, so I've renamed without using "TI" at all.  This also ends up being more consistent with other methods in the class.  For the method above, I renamed it `renderOrgMembersTableRow`.